### PR TITLE
bug/367_null_matching_table

### PR DESCRIPTION
**Pull request by frbattid**
* Fixes issue https://github.com/telefonicaid/fiware-cygnus/issues/367
* 100% unit tests passed
```
Results :
Tests run: 45, Failures: 0, Errors: 0, Skipped: 0
```
* (unofficial) e2e tests passed
* Assignee @fgalan but LGTM from @gtorodelvalle is also required.

### CODE REVIEW

File: src/main/java/com/telefonica/iot/cygnus/interceptors/DestinationExtractor.java [hunk 1]
```diff
@@ -78,6 +78,12 @@ public DestinationExtractor(String matchingTableFile) {
     public void initialize() {
```
Comment (fgalan) [inline src/main/java/com/telefonica/iot/cygnus/interceptors/DestinationExtractor.java]: Maybe an entry in CHANGES_NEXT_RELEASE is missing?

File: src/test/java/com/telefonica/iot/cygnus/interceptors/DestinationExtractorTest.java [hunk 2]
```diff
@@ -1,12 +1,12 @@
 /**
- * Copyright 2014 Telefonica Investigación y Desarrollo, S.A.U
+ * Copyright 2015 Telefonica Investigación y Desarrollo, S.A.U
  *
  * This file is part of fiware-cygnus (FI-WARE project).
  *
- * cosmos-injector is free software: you can redistribute it and/or modify it under the terms of the GNU Affero General
+ * fiware-cygnus is free software: you can redistribute it and/or modify it under the terms of the GNU Affero General
```
Comment (gtorodelvalle) [inline src/test/java/com/telefonica/iot/cygnus/interceptors/DestinationExtractorTest.java]: I noticed that there are many other references to "cosmos-injector" in this branch. Although I guess it is not the most important part of this PR, maybe an issue to fix all these references is needed. I really do not know if it is already created, considered or directly should not be fixed :-) If you would like me to create this issue, please just tell ;) Thanks!

**Comment by gtorodelvalle** [comment]
Nothing relevant to point out from my side but a minor comment :)

File: src/test/java/com/telefonica/iot/cygnus/interceptors/DestinationExtractorTest.java (hunk 2 above)
Comment (frbattid) [inline src/test/java/com/telefonica/iot/cygnus/interceptors/DestinationExtractorTest.java]: Yes, I noticed that :( This is like a plague, there is always some reminiscence from the past... Please, create the issue. Thanks!

Comment (fgalan) [inline src/test/java/com/telefonica/iot/cygnus/interceptors/DestinationExtractorTest.java]: "Replace all" feature in the IDE should helpt to fix this in a single operation ;)

Comment (frbattid) [inline src/test/java/com/telefonica/iot/cygnus/interceptors/DestinationExtractorTest.java]: Yes, the problem is sometimes it is "fiware-connectors", other time it is "orion-connectors", or "cosmos-injector"... I think in the past we played with a lot of different names before choosing "cygnus" :)

File: src/main/java/com/telefonica/iot/cygnus/interceptors/DestinationExtractor.java (hunk 1 above)
Comment (frbattid) [inline src/main/java/com/telefonica/iot/cygnus/interceptors/DestinationExtractor.java]: Fixed in https://github.com/telefonicaid/fiware-cygnus/commit/e2a78c5c2bf7b2142cfc89863c9ea3e7b960de24

File: src/test/java/com/telefonica/iot/cygnus/interceptors/DestinationExtractorTest.java (hunk 2 above)
Comment (frbattid) [inline src/test/java/com/telefonica/iot/cygnus/interceptors/DestinationExtractorTest.java]: Issue created: https://github.com/telefonicaid/fiware-cygnus/issues/371

**Comment by fgalan** [comment]
LGTM

**Pull request merged**